code/style.css

```diff
@@ -1,40 +1,65 @@
 /* Global css variables used for colors */
 :root {
-  --primary: #a259ff;
-  --secondary: #b0a6ff;
+  --primary: white;
+  --secondary: blue;
 }
```

Comment:
The code is so easy to read, tidy and nice. Fantastic!
 Some more comments could make it a bit more personalized.